README.md

```diff
@@ -178,6 +178,14 @@ For Arch Linux you can download `gitoxide` from `community` repository:
 pacman -S gitoxide
 ```
 
+### Download from Exherbo Linux rust repository
+
+For Arch Linux you can download `gitoxide` from [rust](https://gitlab.exherbo.org/exherbo/rust/-/tree/master/packages/dev-scm/gitoxide) repository:
+```
+cave resolve -x repository/rust
+cave resove -x gitoxide
```

Comment:
This is probably a typo.

Reply:
Sorry for that, some checks were missed 😅